.github/workflows/main.yml

```diff
@@ -167,6 +167,15 @@ jobs:
       - name: git checkout
         uses: actions/checkout@v3
 
+      - name: git checkout fuzz seed corpus
+        uses: actions/checkout@v3
+        with:
+          repository: morehouse/lnd-fuzz-seed-corpus
+          path: fuzz-seed-corpus
+
+      - name: rsync fuzz seed corpus
+        run: rsync -a --ignore-existing fuzz-seed-corpus/ ./
```

Comment:
Since the `lnd-fuzz` repo stores the corpora in the proper directory structure (e.g., `lnwire/testdata/fuzz/...`), we don't need to recreate the structure here in the `lnd` repo.  Simply rsycing the `lnd-fuzz` repo into the `lnd` repo will copy the seeds to the right place.